README.rst

```diff
@@ -40,14 +40,57 @@ USAGE
 
 If you want to learn more about coala-bears, its functionality and its usage,
 please take a look at our
-`tutorial <http://coala.readthedocs.org/en/latest/Users/Tutorials/Writing_Bears.html#what-is-a-bear>`__.
+`tutorial <http://coala.readthedocs.org/en/latest/index.html>`__.
```

Comment:
imo i disagree with this link, it gives no information about coala-bears, and this readme is on coala-bears. the first one was better and gave a little more information :)